js/views/composer.js

```diff
@@ -218,13 +255,18 @@ define(function(require) {
 			var bcc = this.$('.bcc');
 			var subject = this.$('.subject');
 
-			message.body = newMessageBody.val();
 			message.to = to.val();
 			message.cc = cc.val();
 			message.bcc = bcc.val();
 			message.subject = subject.val();
 			message.attachments = this.attachments.toJSON();
-
+			if($('.toggle-editor').prop('checked')) {
+				message.type = 'text/plain';
+				message.body = newMessageBody.val();
+			}else{
+				message.type = "text/html"
+				message.body = newMessageBody.trumbowyg('html').replace('<br>&gt;','\n>');
```

Comment:
what are you doing here? I guess if the content is HTML we shouldn't care about '\n's, should we?